samplesheets/assayapps/cytof/plugins.py

```diff
@@ -0,0 +1,136 @@
+"""Assay app plugin for samplesheets"""
+
+from django.conf import settings
+
+# Projectroles dependency
+from samplesheets.plugins import SampleSheetAssayPluginPoint
+from samplesheets.utils import get_last_material_name
+from samplesheets.views import MISC_FILES_COLL
+from samplesheets.rendering import SIMPLE_LINK_TEMPLATE
+
+# Local constants
+APP_NAME = 'samplesheets.assayapps.cytof'
+
+
+class SampleSheetAssayPlugin(SampleSheetAssayPluginPoint):
+    """Plugin for mass cytometry assays in sample sheets"""
+
+    #: Name (used in code and as unique idenfitier)
+    name = 'samplesheets_assay_cytof'
+
+    #: Title
+    title = 'Mass Cytometry Assay plugin'
```

Comment:
It's customary to have these titles in title case and prefixed with `Sample Sheets`, in this case that would be `Sample Sheets Mass Cytometry Assay Plugin`.

Reply:
That's in the description field though, right?